docs/src/manual/installation.md

```diff
@@ -62,7 +62,13 @@ installkernel("Julia nodeps", "--depwarn=no")
 ```
 and a kernel called `Julia nodeps 0.7` (if you are using Julia 0.7)
 will be installed (will show up in your main Jupyter kernel menu) that
-lets you open notebooks with this flag.
+lets you open notebooks with this flag. Note that the default kernel
+that IJulia installs passes the `--project=@` option to Julia, if you
```

Comment:
I think that's `@.`, not `@`, right?

I've made this typo myself… and actually, I don't know what `--project=@` does, if anything.

Reply:
Oops, yes you're quite right :upside_down_face: Fixed in 4d82b4b.